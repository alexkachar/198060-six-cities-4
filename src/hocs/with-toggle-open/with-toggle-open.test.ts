import * as React from 'react';
import renderer from 'react-test-renderer';
import withToggleOpen from './with-toggle-open';

const MockComponent = () => <div />;
const WrappedComponent = withToggleOpen(MockComponent);

it(`withToggleOpen renders correctly`, () => {
  const tree = renderer
    .create(
        <WrappedComponent />, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
