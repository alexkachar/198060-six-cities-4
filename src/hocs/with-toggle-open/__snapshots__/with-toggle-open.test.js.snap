// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withToggleOpen renders correctly 1`] = `<div />`;
